bot/cmd/main.go

```diff
@@ -51,6 +53,13 @@ func main() {
 	go mastodonClient.Run(ctx, cancel, errs)
 	go mastodonClient.Write(ctx)
 
+	bskyClient := bsky.NewClient(
+		cfg.BlueSky.FeedsConfigFile,
+		gsheetClient,
+	)
+
+	bskyClient.Run()
```

Comment:
Looking closer, this might need to be refactored to include the `errs` channel so as to not block the main thread.